coffee_core/src/coffee.rs

```diff
@@ -256,6 +273,14 @@ impl PluginManager for CoffeeManager {
         let plugins = &mut self.config.plugins;
         if let Some(index) = plugins.iter().position(|x| x.name() == plugin) {
             let plugin = plugins[index].clone();
+            let root_path = format!(
+                "{}/{}/plugins/{}",
+                self.config.root_path,
+                self.config.network,
+                plugin.name()
+            );
```

Comment:
this will be `plugin.root_path`, you do not need to rebuild the path each time, the plugin should have this information already